test/src/e2e.long/reward2.test.ts

```diff
@@ -51,149 +51,6 @@ describe("reward2", function() {
         ).to.deep.equal(new U64(50));
     }).timeout(30_000);
 
-    it("alice creates one block and bob creates two blocks in parallel. And then, sync", async function() {
```

Comment:
I think it should be the first commit of this PR.

Reply:
I changed the order

Reply:
I changed the order